t/ansi-test/define-condition.lisp

```diff
@@ -579,6 +579,41 @@
   (notnot-mv (typep #'condition-27/s1 'generic-function))
   t)
 
+;;; Test non-CONDITION supertypes
```

Comment:
Could you move this whole section to `more-tests.lisp`? I'd rather not modify the ANSI-TEST files themselves.

Reply:
:heavy_check_mark: